oml/models/vit_dinov2/extractor.py

```diff
@@ -0,0 +1,123 @@
+from pathlib import Path
+from typing import Optional, Union
+
+import torch
+
+from oml.interfaces.models import IExtractor
+from oml.models.utils import (
+    remove_criterion_in_state_dict,
+    remove_prefix_from_state_dict,
+)
+from oml.models.vit_dinov2.external.hubconf import (  # type: ignore
+    dinov2_vitb14,
+    dinov2_vitb14_reg,
+    dinov2_vitl14,
+    dinov2_vitl14_reg,
+    dinov2_vits14,
+    dinov2_vits14_reg,
+)
+from oml.utils.io import download_checkpoint_one_of
+from oml.utils.misc_torch import normalise
+
+_DINOV2_BASE_URL = "https://dl.fbaipublicfiles.com/dinov2"
+
+
+class ViTExtractor_v2(IExtractor):
+    """
+    The base class for the extractors that follow VisualTransformer architecture.
```

Comment:
Seems like you forgot to remove it

Reply:
Yes, definitely

Reply:
No need to fix because of fcd5cdd6ed4d5d9f2ca4ac9b39afcb4534a357c1